src/test/resources/herrenberg/herrenberg.geojson

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
I don't see a reference to this geojson. What is it needed for?

Reply:
It isn't needed but I find it useful for visualising later which part of the map I have included in the cropped OSM file.

Reply:
Github displays GeoJSON fles on a map: https://github.com/mfdz/OpenTripPlanner/blob/fix-tracks/src/test/resources/herrenberg/herrenberg.geojson

Reply:
Agreed. Thanks.